.circleci/config.yml

```diff
@@ -63,6 +63,7 @@ defaults: &defaults
          pip install lxml
          pip install flake8
          pip install isort
+         pip install redislite
```

Comment:
Not sure you actually need this line if you add `redislite` to `tests_require` in `setup.py`